homework/week-0/AllisonBojarski/TextAdvGame(HWproblem#3)/src/nyc/c4q/alizinha/Fight.java

```diff
@@ -0,0 +1,7 @@
+package nyc.c4q.alizinha;
+/**
+ * Created by c4q-Allison on 5/6/15.
+ */
+public interface Fight
```

Comment:
This is never used. A suggestion for this would be to take your battle simulation in your main and put it into a class that implements some methods that use signatures created here.